pcap/pcap.go

```diff
@@ -89,6 +89,29 @@ int pcap_set_rfmon(pcap_t *p, int rfmon) {
 #define gopacket_time_secs_t time_t
 #define gopacket_time_usecs_t suseconds_t
 #endif
+int pcap_wait(pcap_t *p, int usec) {
+	fd_set fds;
+	int fd;
+	struct timeval tv;
+
+	fd = pcap_get_selectable_fd(p);
```

Comment:
I added separate files for windows/non-windows. From the standard library, there was one example of a `+build !windows` and the filename was `_unix.go` instead of `_other.go`.

In the Windows version, I left the pcap handle in blocking mode -- it should work at least as well as the existing code.